catalog/kubedb/raw/mongodb/mongodb-5.0.23-official.yaml

```diff
@@ -40,4 +40,7 @@ spec:
   ui:
   - name: mongo-ui
     version: v2024.4.27
+  updateConstraints:
+    allowlist:
+    - '> 7.0.0'
```

Comment:
From 5.0.23 to 7.0.8 is possible ? And Not  5.0.23 to 6.0.12   ? 
Does seem right